service/awsconfig/v10/resource/s3bucket/create.go

```diff
@@ -45,6 +45,24 @@ func (r *Resource) ApplyCreateChange(ctx context.Context, obj, createChange inte
 			return microerror.Mask(err)
 		}
 
+		err = r.createAccessLogBucket(ctx, bucketInput)
+		if err != nil {
+			return microerror.Mask(err)
+		}
+
+		_, err = r.clients.S3.PutBucketLogging(&s3.PutBucketLoggingInput{
+			Bucket: aws.String(bucketInput.Name),
+			BucketLoggingStatus: &s3.BucketLoggingStatus{
+				LoggingEnabled: &s3.LoggingEnabled{
+					TargetBucket: aws.String(bucketInput.Name + "-logs"),
+					TargetPrefix: aws.String(key.ClusterID(customObject) + "-access-logs/"),
+				},
+			},
+		})
+		if err != nil {
+			return err
```

Comment:
Error masking is missing here (there are some more occurrences of this).

Reply:
Yeah, I made on purpose because I masked in the caller function. Is it a good practice to do it always?

Reply:
Yep, current convention is masking everywhere.